src/components/Chat/BottomSheetGif.js

```diff
@@ -27,18 +28,10 @@ const BottomSheetGif = React.forwardRef((props, ref) => {
   const [list, setList] = React.useState([]);
   const [isLoading, setIsLoading] = React.useState(true);
 
-  const handleChangeText = (text) => {
-    setSearchText(text);
-  };
-
-  const handleOnClearText = () => {
-    setSearchText('');
-  };
-
-  const onLoad = async () => {
+  const onLoad = async (search) => {
     setIsLoading(true);
     try {
-      const result = await getGifFeatured();
+      const result = await getGifFeatured(search);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onLoad` function has been updated to accept a `search` parameter. This allows the function to be used for both initial loading and subsequent searches. However, error handling seems to be missing here. If the `getGifFeatured(search)` function throws an error, it will not be caught and handled, potentially leading to an unhandled promise rejection.
```diff
- const onLoad = async () => {
+ const onLoad = async (search) => {
    setIsLoading(true);
    try {
      const result = await getGifFeatured(search);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->